pulseapi/profiles/models/categories.py

```diff
@@ -59,3 +63,79 @@ class ProgramYear(models.Model):
 
     def __str__(self):
         return self.value
+
+
+class CohortRecord(models.Model):
+    profile = models.ForeignKey(
+        'profiles.UserProfile',
+        on_delete=models.CASCADE,
+        related_name='cohort_records',
+    )
+
+    program = models.ForeignKey(
+        'profiles.ProgramType',
+        on_delete=models.PROTECT,
+        related_name='profile_cohort_records',
+    )
+
+    year = models.PositiveSmallIntegerField(
+        # TODO: Change to MaxValueValidator with callable when
+        #       we update to Django > v2.2
```

Comment:
do we have an issue filed for updating Django? If not, let's file one and then we can refer to the gh link here, too.

Reply:
https://github.com/MozillaFoundation/mofo-devops/issues/673